### DISCUSSION TITLE
Using different tsai models but get same accuracy

### DISCUSSION THREAD

**Discussion by marcuswang6** [timeseriesAI/tsai#528] [Q&A]
Hi, I was using different model, e.g. FCN, ResNet, MLP, Transformer..., but got the same accuracy with different mcc, f1, precision, recall.
The loss and output of the model were like this:
<img width="774" alt="image" src="https://user-images.githubusercontent.com/41324537/170670010-0d2a5d49-0313-49a9-bfb2-6940e8a0a260.png">
I was wondering what could cause this problem, thanks!

**Comment by vrodriguezf** [comment]
I have seen that in some of my experiments too, in my experience the choice of the archtecture is not awalys relevant to the task at hand.

**Reply by marcuswang6** [reply]
> I have seen that in some of my experiments too, in my experience the choice of the archtecture is not awalys relevant to the task at hand.

Thanks for your reply!
How do you solve this problem then?

**Reply by vrodriguezf** [reply]
It depends on the data and the task. Sometimes the key lied in dealing with the imbalanced nature of the data, others it was about data augmentation to solve for the lack of training data...sometimes the things that you change in the data affect much more the results than the architecture, once you have set up a good one such as InceptionTime or TSiT
